data/delegates.json

```diff
@@ -148,6 +148,22 @@
     "required": true
   },
   {
+    "delegateName": "4fryn",
+    "delegateAddress": "12632212355488748579L",
+    "proposal": "https://forum.lisk.io/viewtopic.php?t=1560",
+    "githubUsername": "4fryn",
+    "poolPercentage": {
+      "P": "80",
+      "D": "15",
+      "O": "5"
+    },
+    "affiliation": "Freelance",
+    "liskChat": "4fryn",
+    "pool": null,
```

Comment:
I see you set up pool percentage to 80% but haven't provide a URL to your pool yet, does that mean you don't have a website yet? If you do have a web page for your pool, please add it.